transport.go

```diff
@@ -90,39 +91,40 @@ func (t *transport) IsConnected() bool {
 	}
 }
 
-func (t *transport) Run() error {
-	if !t.IsConnected() {
```

Comment:
Actually, I think I did intend to drop this. Either this is the first call to ReceiveFrames(), in which case IsConnected is always true (since stopCh is closed in receiveFrames()) or this is the second time through, in which case we return a different error.